airflow/providers/jdbc/hooks/jdbc.py

```diff
@@ -152,7 +153,8 @@ def set_autocommit(self, conn: jaydebeapi.Connection, autocommit: bool) -> None:
         :param conn: The connection.
         :param autocommit: The connection's autocommit setting.
         """
-        conn.jconn.setAutoCommit(autocommit)
+        with suppress(jaydebeapi.Error):
+            conn.jconn.setAutoCommit(autocommit)
```

Comment:
Should we print a warning instead? That seems like pretty unexpected state and might be difficult to debug problems when someone expects autocommit to be effective.

Reply:
Yes indeed that would be better will do that, I assumed the suppress would already do that but that's not the case indeed.